### DISCUSSION TITLE
How to use quartznet with autofac in .net framework 4.8

### DISCUSSION THREAD

**Discussion by ramkumarkrk** [quartznet/quartznet#914] [General]
How to use **quartznet** with **autofac** in **.net framework 4.8**? When I tried to inject the service in the job constructor the job is not getting executed.

**Comment by lahma** [comment]
First you need custom job factory and I believe there's a [pre-made one for Autofac](https://github.com/alphacloud/Autofac.Extras.Quartz). It seems to support both full framework and .NET Core.

Their repository contains samples so they should be quite good starting point. Quartz project doesn't support other containers that the Microsoft one.
